.binder/requirements.txt

```diff
@@ -0,0 +1,7 @@
+--find-links https://sklearn-nightly.scdn8.secure.raxcdn.com scikit-learn
+matplotlib
+scikit-image
+pandas
+sphinx-gallery
+scikit-learn>=0.22.dev0
```

Comment:
For my education what does `--find-links https://sklearn-nightly.scdn8.secure.raxcdn.com scikit-learn` and `scikit-learn>=0.22.dev0` do? Is the result that you always install the latest nightly?

Reply:
Yes. Maybe using the `--pre` flag and the `--find-links URL` flag would be enough (instead of putting the version constraint for scikit-learn).

https://scikit-learn.org/stable/developers/advanced_installation.html#installing-nightly-builds